scaleway/service.go

```diff
@@ -44,6 +44,11 @@ func getSessionConfig(ctx context.Context, d *plugin.QueryData) (*scw.Client, er
 		secretKey = *scalewayConfig.SecretKey
 	}
 
+	// No Organization Id
+	if scalewayConfig.OrganizationID == nil {
```

Comment:
@jplanckeel What Scaleway resources require this organization ID beside users and API keys? Do any of the existing tables require it?

Up until now, we haven't had any tables that required it, so adding it in now as a required config arg would be a breaking change, i.e., anyone who uses the Scaleway plugin today would suddenly be unable to until they added the `organization_id` config arg.

In some plugins, we add optional config args and then note in the appropriate table docs which optional config args are required, e.g., https://hub.steampipe.io/plugins/turbot/hibp#configuration and https://hub.steampipe.io/plugins/turbot/hibp/tables/hibp_paste.